docs/overview.rst

```diff
@@ -131,6 +136,21 @@ admin`_) can see an overview of logged changes to the organization, the
 organization's projects, and other such data, each provided with a timestamp.
 
 
+Organization Billing
+--------------------
+
+This tab shows the billing overview. If you have signed up through the
+recommended route, using a credit card to pay for your cluster, the overview
+of your chosen means of payment will be shown here.
+
+.. image:: _assets/img/organization-billing.png
+   :alt: Cloud Console organization billing tab
+
+In case you use multiple cards, a default card can be set. Cards can be deleted
+from the list by using the dots icon on the far right of the card listing.
+Click the *Add payment method* button at the top right to add a new card.
```

Comment:
```suggestion
012345678901234567890123456789012345678901234567890123456789
This tab shows the billing overview. If you signed up with a credit card for your 
cluster (the recommended route), a payment method overview will be shown
here.

.. image:: _assets/img/organization-billing.png
   :alt: Cloud Console organization billing tab

In case you use multiple cards, a default card can be set. Cards can be deleted
from the list by using the dots icon to the right of the card listing.
Click the *Add payment method* button at the top right to add a new card.
```

Reply:
012345678901234567890123456789012345678901234567890123456789? 🧐🤷‍♂️

Reply:
oh darnit, sorry, I meant to remove that ... it's a line length guide.

Reply:
i removed it now :)

Reply:
Ah I figured, haha. No probs